components/html-block/test/html-block.visual-diff.html

```diff
@@ -26,64 +26,60 @@
 	</div>
 
 	<div class="visual-diff">
-		<d2l-html-block id="typography">
```

Comment:
These were all subtly different from the demo case and so I can almost guarantee some of these will be changed slightly. I'll take a peek if the visual diff tests fail and either update the diffs if we've lost useful test info, or merge the new goldens if it's just slightly different content/styling.